owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -316,12 +317,22 @@ public void onRemoteOperationFinish(RemoteOperation operation, RemoteOperationRe
         }
     }
 
-    protected void showRequestAccountChangeNotice() {
-        Snackbar.make(findViewById(android.R.id.content), R.string.auth_failure_snackbar, Snackbar.LENGTH_INDEFINITE)
-                .setAction(R.string.auth_failure_snackbar_action, v ->
-                        startActivity(
-                                new Intent(FileActivity.this, ManageAccountsActivity.class)))
-                .show();
+    protected void showRequestAccountChangeNotice(String errorMessage, boolean mustChange) {
```

Comment:
What about initializing `mustChange` to false (boolean mustChange = false) and set it to true just when needed?

Reply:
Sure, that would be great, but unfortunately this is not kotlin yet 😢

Reply:
Ok, let's postpone it then